contents/blog/first-ops-hire.mdx

```diff
@@ -0,0 +1,73 @@
+---
+date: 2023-09-07
+title: Making your first startup ops hire - what founders should look for
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - charles-cook
+featuredImage: ../images/blog/posthog-blog-image.png
+featuredImageType: full
+category: Startups
+tags:
+  - Guides
+---
+
+I've [previously written](https://posthog.com/blog/early-stage-startup-hiring-strategy) about startup hiring strategy more generally. This article covers a specific role - what to look out for when making your first ops hire into your startup, aka probably your first non-engineer. Usually this person is hire number 5-10, and they will take over stuff that one of the cofounders has been doing up to that point.
+
+Your first ops person will have a disproportionate impact on your company culture, because they do work that affects everyone on the team personally, so it's worth getting this stuff right.
+
+You want your first ops hire to be:
+- Extremely optimistic
+- Willing to eat a lot of shit
+- Utterly dependable
```

Comment:
```suggestion
It's an important hire because your first ops person will have a disproportionate impact on your company culture. They'll do work that affects everyone on the team personally, and hopefully ensure the founders can focus on [finding product-market fit](/blog/product-market-fit-game) and making the company a success. 

With that in mind, these are the three attributes your first ops hire needs...
```